internal/pkg/deploy/cloudformation/stack/backend_svc.go

```diff
@@ -212,6 +239,18 @@ func (s *BackendService) httpLoadBalancerTarget() (targetContainer *string, targ
 		targetPort = s.manifest.Sidecars[aws.StringValue(targetContainer)].Port
 	}
 
+	// Route load balancer traffic to the target_port if mentioned.
+	if s.manifest.RoutingRule.TargetPort != nil {
+		targetPort = aws.String(strconv.Itoa(aws.IntValue(s.manifest.RoutingRule.TargetPort)))
+		exposedPorts, err := s.manifest.ExposedPorts()
+		if err == nil {
```

Comment:
I think it is safe to sqallow error because the `ExposedPorts()` is throwing the same error that [validExposedPort](https://github.com/aws/copilot-cli/pull/4192/files#diff-5fc13795122fe030799a09ef956a8b66cec3d40bf1a81266323edbf3626b5685R1732) is throwing. So I think if error has to come then it will be surfaced during validation phase itself.